subsys/bootloader/c_runtime_setup/Kconfig

```diff
@@ -4,17 +4,9 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
-choice
-	prompt "C runtime setup selection"
-	default SB_C_RUNTIME_SETUP_VARIANT_MDK
-	config SB_C_RUNTIME_SETUP_VARIANT_CUSTOM
-		bool "Custom C runtime setup"
-	config SB_C_RUNTIME_SETUP_VARIANT_ZEPHYR
-		bool "Zephyr C runtime setup"
-	config SB_C_RUNTIME_SETUP_VARIANT_MDK
-		bool "MDK C runtime setup"
-		select SB_VENDOR_SYSTEM_INIT
-endchoice
+config SB_C_RUNTIME_SETUP_VARIANT_ZEPHYR
+	bool "Zephyr C runtime setup"
+	default y
```

Comment:
What happens if this is set to 'n'?

Reply:
Compile error, I'll just remove the prompt

Reply:
Done